docs/data/osdf/install-cache-container.md

```diff
@@ -0,0 +1,347 @@
+
+
+Preparing for Initial Startup
+-----------------------------
+
+1.  The cache identifies itself to the federation via public key authentication;
+before starting the cache for the first time, it is recommended to generate a keypair.
+Download the Pelican client from <https://github.com/PelicanPlatform/pelican/releases> and run:
+
+        :::console
+        root@host$ cd /etc/pelican
+        root@host$ pelican generate keygen
+
+
+    The newly created files, `issuer.jwk` and `issuer-pub.jwks` are the private and public keys, respectively.
+
+1.  **Save these files**; if you lose the `issuer.jwk`, your cache will need to be re-approved.
+
+
+Running a Cache
+---------------
+
+This section describes how the cache may be run using the Docker command-line.
+For production, we recommend using a container orchestration service such as
+[Docker Compose](https://docs.docker.com/compose/),
+or [Kubernetes](https://kubernetes.io/).
+
+This table provides a reminder of the parameters that will be used in the examples:
+
+| Parameter             | Example value                    | Description                                                                             | 
+|-----------------------|----------------------------------|-----------------------------------------------------------------------------------------|
+| `CACHE_HOST_PORT`     | `8443`                           | The host port used for data transfer via HTTP(S)                                        | 
+| `WEB_INTERFACE_PORT`  | `8444`                           | The host port used for service management via the web interface                         | 
+| `ISSUER_JWK`          | `/etc/pelican/issuer.jwk`        | The private key generated by `pelican generate keygen`                                  | 
+| `HOST_CERT_CHAIN`     | `/etc/pelican/hostcert.pem`      | The host certificate, with full chain                                                   | 
+| `HOST_KEY`            | `/etc/pelican/hostkey.pem`       | The private key matching the host certificate                                           | 
+| `CACHE_CONFIG_FILE`   | `/etc/pelican/cache-config.yaml` | The YAML file containing various cache settings                                         | 
+| `CACHE_PARTITION`     | `/mnt/cache`                     | The partition to be used for storing cached data (single partition setup only)          | 
+| `NAMESPACE_PARTITION` | `/mnt/cache-namespace`           | The partition to be used for storing namespace information (multi-partition setup only) | 
+| `METADATA_PARTITION`  | `/mnt/cache-meta`                | The partition to be used for storing metadata (multi-partition setup only)              | 
+| `DATA_PARTITION`      | `/mnt/cache-data`                | The partition to be used for storing cached file contents (multi-partition setup only)  | 
+
+
+### Single host partition
+
+A cache may be configured to use a single host partition only;
```

Comment:
Is it a single "host partition" or a "single host" partition? I think this can be lightly clarified in the first sentence here by rearranging some of the words

Reply:
I just got rid of the word "host" since it's superfluous.